assets/css/frontend.scss

```diff
@@ -370,6 +370,88 @@ a.sensei-certificate-link {
   }
 }
 
+.sensei-quiz-pagination {
+	display: flex;
+	flex-wrap: wrap;
+	align-items: center;
+	gap: 1.5em;
+	margin: 1.618em 0;
+
+	@media (max-width: 667px) {
```

Comment:
Could we use a mobile first approach here and add base styles for mobile, then add more styles on top of that for wider viewports (i.e. switch from `max-width` to `min-width` media queries)?

Reply:
Good suggestion. Using mobile first should be less complicated in general. Fixed in 8ba389fc02e281504f468552b704fe16bf441da6.